public/apps/configuration/utils/tenant-utils.tsx

```diff
@@ -179,6 +179,33 @@ export function transformRoleTenantPermissions(
   }));
 }
 
+export function getNamespacesToRegister(accountInfo: any) {
+  const tenants = accountInfo.tenants || {};
```

Comment:
@cliu123 I noticed when testing this change that the following data structure comes out as:

```
{
    "global_tenant": true,
    "admin_tenant": true,
    "admin": true
}
```

Will the values in this map of tenants ever contain `false`? If so, should this skip over the tenants where the value in this object is `false`?

Reply:
@cwperks You might have logged in as admin, then you are expected to have all tenant permissions. If you create a role, grant permission to a specific tenant, create a user,  map the role to the user, and log in as the user, then you'll see the difference.